src/pages/Coach/Chat.tsx

```
@@ -98,18 +99,30 @@ export default function CoachChatPage() {
       if (demo) demoToast();
       return;
     }
-    const trimmed = input.trim();
-    if (!trimmed) return;
+    const sanitized = input
+      .replace(/[\u0000-\u001F\u007F]/g, " ")
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)